scripts/core/Utils.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+#define _CRT_SECURE_NO_WARNINGS
+#include <iostream>
+#include <string>
+#include <chrono>
+#include <iomanip>
+#include <sstream>
+#include <ctime>
+#include <windows.h>
+
+#undef ERROR // conflicts with our ERROR enum. we can either change our enum or just keep this undefed. not like we need it. yet.
+class Logger {
+public:
+    enum class Level {
+        INFO,
+        SUCCESS,
+        WARN,
+        ERROR
```

Comment:
It is better to instantly classify some of the logging outputs as debug, so that we can easily turn them off when running as production code.
```suggestion
        ERROR,
        DEBUG
```